pkgs/by-name/gr/graplang/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl
+, flex, bison }:
+
+stdenv.mkDerivation {
+  pname = "graplang";
+  version = "1.46";
+
+  src = fetchurl {
+    url = "https://www.lunabase.org/~faber/Vault/software/grap/grap-1.46.tar.gz";
+    hash = "sha512-7n+jLANU/x+wGrpjwYAnf45fQ5M91SwraiCbvUKe6XhWtilhGoT2yTlLkPlTihETTkizLyssW5gj5gbwNHaooA==";
+  };
+
+  nativeBuildInputs = [ flex bison ];
+
+  meta = with lib; {
+    description = "Language for typesetting graphs";
+    longDescription = ''
+      Grap is an Expressive language for describing graphs and incorporating
+      them in typeset documents.  It is implemented as a preprocessor to
+      Kernigan's pic language for describing languages, so any system that can
+      use pic can use grap. For sure, TeX and groff can use it.
+    '';
+    homepage = "https://www.lunabase.org/~faber/Vault/software/grap/";
+    changelog = "https://github.com/snorerot13/grap/blob/master/CHANGES";
```

Comment:
Why not to use fetchFromGitHub?

Reply:
The GitHub repo has neither [releases](https://github.com/snorerot13/grap/releases) nor [tags](https://github.com/snorerot13/grap/tags) opposed to the [homepage](https://www.lunabase.org/~faber/Vault/software/grap/) which provides a link to the versioned sources.